content/themis/languages/python/installation.md

```diff
@@ -75,6 +75,58 @@ Once PyThemis is installed, you can [try out code examples](../examples/).
 If the stable package version does not suit your needs,
 you can manually build and install the latest version of Themis from source code.
 
+ 1. [Build and install Themis Core library](/themis/installation/installation-from-sources/)
+    into your system.
+
+ 2. Create a "wheel" package
+
+    <!-- FIXME: For some reason (missing install tools in venv?) the build fails when venv is activated,
```

Comment:
🤔

Reply:
Made improvement in Makefile that ignores venv if it was activated, so building the wheel works now in such scenario. As long as there is python3-setuptools installed in system. Alternatively we could remove that hack and just say in docs "hey, you need no venv when packing PyThemis"

Reply:
the easier it is for the user — the better.
@Lagovas wdyt?